### PR TITLE
[docker] Build Python3.6 & Python3.8 Docker Images

### PR DESCRIPTION
**Pull request by ijrsvt**
## Why are these changes needed?
* Build Docker images for Py36, Py38!
* NOTE: This is a separate job because if we run on just one job, we run out of storage!

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ci/travis/build-docker-images.py [hunk 1]
```diff
@@ -81,64 +91,76 @@ def _docker_affected():
...
-        print("BUILT: ", tagged_name)
-        built_images.append(tagged_name)
+                # NOTE(ilr) This is a bit of an abuse of the name "GPU"
+                build_args["GPU"] = f"{py_name}{gpu}"
+
+            if image_name in ["ray", "ray-deps"]:
+                build_args["WHEEL_PATH"] = f".whl/{_get_wheel_name(build_args['PYTHON_MINOR_VERSION'])}"
+
+
+
```
Comment (simon-mo) [inline ci/travis/build-docker-images.py:112]: extra vertical whitespace

**Review by simon-mo — APPROVED** [review]
nice work! let's make sure to change the names a bit
s/ORIGINAL/PY37
s/NEW/PY36_PY38

File: .travis.yml [hunk 2]
```diff
@@ -209,10 +209,31 @@ matrix:
...
         - python -m pip install docker
-        - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then python $TRAVIS_BUILD_DIR/ci/travis/build-docker-images.py; fi
+        - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then python $TRAVIS_BUILD_DIR/ci/travis/build-docker-images.py ORIGINAL; fi
         - bash ./java/build-jar-multiplatform.sh linux
       cache: false
 
+
+    # Build Py36 & Py38 Docker Images
+    - os: linux
+      env:
```
[the remark this answers is not in the document]
Comment (ijrsvt) [reply .travis.yml:219]: Definitely will!

**Comment by simon-mo** [comment]
```
0.04s$ if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then python $TRAVIS_BUILD_DIR/ci/travis/build-docker-images.py PY37; fi
  File "/home/travis/build/ray-project/ray/ci/travis/build-docker-images.py", line 60
    f"{_get_root_dir()}/.whl/*{PYTHON_WHL_VERSION}
                                                 ^
SyntaxError: EOL while scanning string literal
```

**Comment by simon-mo** [comment]
```
The command "bash miniconda3.sh -b -p "$HOME/miniconda3"" exited with 0.
0.68s$ conda install -y python=3.7.6
conda: command not found
The command "conda install -y python=3.7.6" exited with 127.
```
conda not in path?

**Pull request merged**